packages/payment-processor/test/payment/erc-20-private-payment-hinkal.test.ts

```
@@ -0,0 +1,149 @@
+import { RequestNetwork, Types, Utils } from '@requestnetwork/request-client.js';
+import { CurrencyTypes } from '@requestnetwork/types';
+import { EthereumPrivateKeySignatureProvider } from '@requestnetwork/epk-signature';
+import {
+  hinkal,
+  payPrivateErc20FeeProxyRequestFromHinkal,
+  payPrivateErc20ProxyRequestFromHinkal,
+} from '../../src/payment/erc-20-private-payment-hinkal';
+import { getErc20Balance } from '../../src/payment/erc20';
+
+// Constants to configure the tests
+export const currencyAddress = '0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913'; // USDC
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)